src/extensions/base/BaseSchema/BaseSchemaSpecs/index.ts

```diff
@@ -62,9 +63,16 @@ export const BaseSchemaSpecs: ExtensionAuto<BaseSchemaSpecsOptions> = (builder,
                     : undefined,
             },
             fromMd: {tokenSpec: {name: BaseNode.Paragraph, type: 'block'}},
-            toMd: (state, node) => {
-                state.renderInline(node);
-                state.closeBlock(node);
+            toMd: (state, node, parent) => {
+                if (opts.allowEmptyRows && !node.content.size) {
+                    const isParentEmpty = parent.content.size / parent.content.childCount === 2;
```

Comment:
add comment plz

Reply:
There are nodes (for example, iframe) that contain size === 1, we suggest changing the logic to bypass the nodes